tests/base/test_cyl.py

```diff
@@ -608,22 +618,22 @@ def test_nE(self):
         self.assertTrue(self.mesh.nEy == 18)
         self.assertTrue(np.all(self.mesh.vnEy == [3, 2, 3]))
         self.assertTrue(self.mesh.nEz == 12 + 2)
-        self.assertTrue(self.mesh.vnEz is None)
+        self.assertTrue(np.all(self.mesh.vnEz == [4, 2, 2]))
```

Comment:
This is only for the 3D one though correct?

For the symmetric one, there should still be no vertical edges.